### DISCUSSION TITLE
How to hide a folder or a particular file from view?

### DISCUSSION THREAD

**Discussion by anand-singh** [planet42/Laika#248] [Q&A] [answered]
I have a case to hide some files from the view but we need to keep them for developer references.

**Comment by jenshalm** [comment, marked as answer]
I think you are looking for https://planet42.github.io/Laika/0.18/03-preparing-content/01-directory-structure.html#limiting-the-output-formats

**Reply by anand-singh** [reply]
Thanks :)
